Description: Link to libm to prevent FTBFS with --no-add-needed.
Author: Evgeni Golov <evgeni@debian.org>
Bug-Debian: http://bugs.debian.org/615775

--- slimevolley-2.4.2.orig/CMakeLists.txt
+++ slimevolley-2.4.2/CMakeLists.txt
@@ -42,6 +42,7 @@ LINK_LIBRARIES(
 	${SDL_LIBRARY}
 	${SDLTTF_LIBRARY}
 	${SDLIMAGE_LIBRARY}
+	m
 )
 
 INCLUDE(FindGettext.cmake)
